simple-hello-world-example/README.md

```diff
@@ -1,42 +1,32 @@
-# simple-python-webserver-blueprint
+# Simple Hello-World Example
 
-The blueprint installs a webserver on your local machine. It is supported on Linux and most *nix machines.
+This blueprint installs a web-server on the Cloudify Manager machine. It is supported on Linux and most *nix machines.
```

Comment:
web-server? or web server?

Reply:
According to the results on google, it should be "web server", so I think "web-server" should be good.